docs/_getting-started-macos-ios.md

```diff
@@ -19,6 +19,31 @@ If you have already installed Node on your system, make sure it is Node 14 or ne
 
 [Watchman](https://facebook.github.io/watchman) is a tool by Facebook for watching changes in the filesystem. It is highly recommended you install it for better performance.
 
+### Ruby
+
+React Native uses a `.ruby-version` file to make sure that your version of Ruby is aligned with what is needed. Currently, macOS 12.5.1 is shipped with Ruby 2.6.8, which is not what is required by React Native. Our suggestion is to install a Ruby version manager (RVM) and to install the proper version of Ruby in your system.
+
+Some common RVM are:
+
+- [rbenv](https://github.com/rbenv/rbenv)
+- [RVM](https://rvm.io/)
+- [chruby](https://github.com/postmodern/chruby)
+- [asdf-vm](https://github.com/asdf-vm) with the [asdf-ruby](https://github.com/asdf-vm/asdf-ruby) plugin
+
+To check what is your current version of Ruby, you can run this command:
+
+```
+ruby --version
+```
+
+React Native uses [this version](https://github.com/facebook/react-native/blob/main/template/_ruby-version) of Ruby.
```

Comment:
```suggestion
React Native uses [this version](https://github.com/facebook/react-native/blob/main/template/_ruby-version) of Ruby. You can also find which version your specific project needs in the `.ruby-version` file at root of your RN project.
```
(since link will always point at main)